proposals/0002-explicit-fixity-(~).md

```diff
@@ -0,0 +1,55 @@
+---
+author: Rinat Stryungis
+date-accepted: ""
+ticket-url: "https://gitlab.haskell.org/ghc/ghc/-/issues/18252"
+implemented: "Yes"
+---
+
+This proposal is [discussed at this pull request](https://github.com/haskell-core/core-libraries-proposals/pull/0>).
+**After creating the pull request, edit this file again, update the number in
+the link, and delete this bold sentence.**
+
+# Proposal title
+
+Added explicit fixity for `(~)` and `(~~)`
+
+## Motivation
+
+It solves the issue by adding an explicit fixity for the `(~)` type operator.
```

Comment:
In the motivation section, you need to state what the issue is, not only its solution. In this case, the issue is demonstrated by the following GHCi session:

```
Prelude Data.Type.Equality GHC.TypeLits> :kind! 5 + 5 :~: 10
5 + 5 :~: 10 :: *
= 10 :~: 10
Prelude Data.Type.Equality GHC.TypeLits> :kind! 5 + 5 ~ 10

<interactive>:1:5: error:
    • Expected kind ‘Nat’, but ‘5 ~ 10’ has kind ‘Constraint’
    • In the second argument of ‘(+)’, namely ‘5 ~ 10’
      In the type ‘5 + 5 ~ 10’
```

One would expect `5 + 5 ~ 10` to work, but it doesn't. This is basically what [`#18252`](https://gitlab.haskell.org/ghc/ghc/-/issues/18252) is saying, but readers of the proposal should be able to understand the issue without reading any tickets (although a link to the ticket would be also useful, as additional information).

Reply:
Ok. About the ticket - there is a ticket URL field fo that.